uncontrast_studies/parsers/stimulus_parser.py

```diff
@@ -250,17 +324,18 @@ def resolve_uncon_target_stimuli(item: dict, index: str):
     stimuli_category_data = clean_list_from_data(item["Stimuli Category 2"])
     stimuli_sub_category_data = clean_list_from_data(item["Stimuli Sub-category 2"])
 
+    if stimuli_category_data == ["missing"] or stimuli_sub_category_data == ["missing"]:
```

Comment:
no case just one of the categories is missing but other not?

Reply:
not for target stimulus